### ISSUE TITLE
Incompatibility with Immersive Portals

### ISSUE THREAD

**Issue by Reelie** [Pokecube-Development/Pokecube-Issues-and-Wiki#258]
#### Issue Description:
The mod [Immersive Portals](https://www.curseforge.com/minecraft/mc-mods/immersive-portals-mod) has an incompatibility with PokeCube. Pokemobs will have stange behavior when crossing the Nether portal if they are sent out.
Mod Link: https://www.curseforge.com/minecraft/mc-mods/immersive-portals-mod

#### What happens:

When changing dimension, if a Pokemob is out of its Pokecube, it will disappear and will no longer be interactable. We keep seeing log message showing the Pokecube is fighting with nearby mobs, but that is all; we can't retrieve it. The only way to get back the conrol of the pokemob is hoping that it will die in some way, then it will return in the PC.

#### What you expected to happen:

To have the normal behavior of a pokemob. Not disappearing and not loosing its control.

#### Steps to reproduce:

1. Install Immersive Portals
2. Build a Nether Portal
3. Send out a Pokemob
4. Enter in the Nether

____
#### Affected Versions (Do *not* use "latest"): Replace with a list of all mods you have in.

- Pokecube AIO: 2.0.7
- Minecraft: 1.14.4
- Forge: 28.2.3

**Comment by Thutmose** [comment]
compatibility with that mod probably won't go very well, as it seems to have converted to a fabric mod instead of a forge mod, so unless you can get an updated version of for forge, I can't really do any compatibility related fixes

**Comment by WenXin20** [comment]
The forge version of the mod is actually at https://www.curseforge.com/minecraft/mc-mods/immersive-portals-for-forge

**Comment by Reelie** [comment]
> The forge version of the mod is actually at https://www.curseforge.com/minecraft/mc-mods/immersive-portals-for-forge

Indeed. That is the one I used. I guess I didn't link the right version :o

**Comment by Thutmose** [comment]
ahh, the original link was to the fabric version, which does have an old forge download, so I thought that was what was used
